onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -515,18 +515,30 @@ static Status HandleAutoPad(const Shape& input_shape,
 static Status GetBinaryOpQuantizationScaleAndZeroPoint(
     const ModelBuilder& model_builder, const Node& node,
     float& a_scale, float& b_scale, float& y_scale,
-    int32_t& a_zero_point, int32_t& b_zero_point, int32_t& y_zero_point) ORT_MUST_USE_RESULT;
+    int32_t& a_zero_point, int32_t& b_zero_point, int32_t& y_zero_point, std::unique_ptr<ConstNodesToOptimize>& qdq_node_group) ORT_MUST_USE_RESULT;
```

Comment:
>ORT_MUST_USE_RESULT [](http://example.com/codeflow?start=128&length=19)

fyi: we don't need ORT_MUST_USE_RESULT for Status returning functions anymore since Status is [[nodiscard]]